frontend/components/LabeledBtn.vue

```diff
@@ -1,3 +1,4 @@
+
 <template>
   <a
```

Comment:
This `<a>` tag should be given an `aria-label` attribute that has `alternateText` assigned to it 😊